osu.Game/Scoring/ScoreRank.cs

```diff
@@ -11,6 +11,11 @@ namespace osu.Game.Scoring
 {
     public enum ScoreRank
     {
+        // TODO: reconsider changing later on
+        [LocalisableDescription(typeof(BeatmapsStrings), nameof(BeatmapsStrings.RankD))]
```

Comment:
Why is this even added? I checked source, `LocalisableDescription` [will gracefully go back to `Description` if the localisable variant isn't found](https://github.com/ppy/osu-framework/blob/2d4142c5433492868f1ba7a2f670afeb84a9e906/osu.Framework/Extensions/ExtensionMethods.cs#L197-L199). So not adding this attribute at all would have been less wrong than doing this.

Reply:
I'm sorry that I didn't read the framwork code carefully.
So just remove this attribute and wait for RankF string added to osu.Resource?